src/components/Guidance.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { actionCreators as guidedActionCreators } from '../ducks/modules/guidance';
+
+class Guided extends Component {
+  static propTypes = {
+    contentId: PropTypes.string.isRequired,
+    setGuidance: PropTypes.func.isRequired,
+    clearGuidance: PropTypes.func.isRequired,
+    children: PropTypes.node.isRequired,
+  };
+
+  handleMouseEnter = () => {
+    this.props.setGuidance(this.props.contentId);
+  }
+
+  handleMouseLeave = () => {
```

Comment:
For form field-specific guidance, would this approach allow for focus/blur setting/clearing of guidance as well? (Seems like it...) Not needed in this PR, but would be nice to have for keyboard navigation of form.

Reply:
That's a cool idea, I might try adding that functionality with a demo to the PR

Reply:
So after a bit of experimentation with this I think it needs to include some kind of history.

So when you focus an element, then mouse over something else, when coming back to the element it can reset to the focussed guidance.

Will make this a separate issue.